src/components/addTodoForm/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+import { addNewTodo, isAdding } from "redux/actions"
+import ImageButton from "components/buttons/imageButton"
+import { customSanitize } from "utils/sanitizeInput"
+import { DELETE_BUTTON_ALT, IMAGE_PATHS } from "utils/constants"
+import "components/addTodoForm/form.scss"
+
+function AddTodoForm() {
+  const dispatch = useDispatch()
+  const [taskTitle, setTaskTile] = useState("")
+  const [isInputFieldEmpty, setIsInputFieldEmpty] = useState(false)
+
+  function handleAddTodo(event) {
+    event.preventDefault()
+    if (taskTitle.trim() === "") {
+      setTaskTile("")
+      setIsInputFieldEmpty(true)
+    } else {
+      const satitizedText = customSanitize(taskTitle.trim())
```

Comment:
Instead of trimming the text here, can we trim the string in `utils/sanitizeInput`